src/test/run-make-fulldeps/print-rustc-path/Makefile

```diff
@@ -0,0 +1,4 @@
+-include ../tools.mk
+
+all:
+	$(RUSTC) --print rustc-path | $(CGREP) bin/rustc
```

Comment:
`run-make-fulldeps` -> `run-make`
I don't think this requires fulldeps.

Reply:
Fair; I just copied `--print cfg`... though since this requires starting up a compiler session, it might? May as well try it... (On mobile atm but will do so when I get back.)